### ISSUE TITLE
Pause commands aren't received and/or passed to Spotify from HomePod mini or Apple TV

### ISSUE THREAD

**Issue by nano9g** [philippe44/SpotConnect#15]
When I tap the top of the HomePod mini, I expect the stream to pause. However, SpotConnect currently just keeps playing. (The playing indicator light turns off until the next track starts.)

For what it's worth, volume up/down *is* handled correctly.

Please let me know if you need any logs or other information. Thanks!

**Comment by philippe44** [comment]
Yes, that woudl be worth a log because "pause" command from remote should be handled

**Comment by philippe44** [comment]
I'm a bit confused about the "states" and the logs. What happens if you just do a play and do not touch the HomePod? What type of log do you have?

**Comment by philippe44** [comment]
> Hopefully that clarifies things, but let me know if you still have questions. 🙂
>
It does but that's surprising to see a few fails/log errors when just trying to play. Do you have other devices like an AirPort express? What are logs on your ATV?

**Comment by philippe44** [comment]
> > It does but that's surprising to see a few fails/log errors when just trying to play.
> 
> Ah, no, that log exerpt is from the one time I tried to pause and Siri activated. Normally there aren't errors when I just start playing.
>
Ok, so just to be 100% when playback in normal case, there is nothing weird in the log? Sorry, this is confusing 😄

**Comment by philippe44** [comment]
Version 0.1.4 should address the Aether Cone play/pause

**Comment by philippe44** [comment]
You're right indeed, I've been a bit careless with that part of the code where there was a deadlock due to use of recursive mutex (if that sort of things rings a bell for you). Should be fine in 0.1.5
